tests/config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
In running the tests on Windows, I noticed a strange phenomenon - this file, config.json, will change its permissions from 644 to 755. I don't see where it is referenced in any of the test configuration, so I assume that it is a native configuration file read by pouchDB. Could you add a test or markdown file config.txt or config.md alongside it in this directory explaining what this file is and how it is read, and warning users of the permissions change?

Thanks